### PR TITLE
CMake fixes:

### PR DESCRIPTION
**Pull request by ihrivnac**
Added FindVGM.cmake to amend the target provided by the VGM installation)

### CODE REVIEW

File: dependencies/FindVGM.cmake [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# use the VGMConfig.cmake provided by the VGM installation but
+# amend the target VGM::XmlVGM with the include  and link directories
+
+find_package(VGM NO_MODULE)
+if(NOT VGM_FOUND)
+  return()
+endif()
+
+# VGM uses namespace since 4.7
+message(STATUS ${VGM_VERSION})
```
Comment (aphecetche) [inline dependencies/FindVGM.cmake:20]: is the message intended or a leftover from the testing phase ?

Comment (ihrivnac) [reply dependencies/FindVGM.cmake:20]: This was for debugging; I will remove it.

**Comment by aphecetche** [comment]
@ihrivnac I guess we hit a problem on CC7. Yesterday I wondered why in the first place I've put the LD_LIBRARY_PATH in the test-root-macros but could not remember. I think that is the reason... Might require some investigation to see if / how we can deal properly both with Mac and CC7 but meanwhile I would propose to remove the change in test-root-macro from this PR and just keep the FindVGM one, to get the other (alidist) PR go through. What do you think ?

**Comment by ihrivnac** [comment]
@aphecetche , thank you. I update the brach.

**Pull request merged**